### ISSUE TITLE
Inequivalent suggestion to convert preg_match to strpos

### ISSUE THREAD

**Issue by JasonTheAdams** [kalessil/phpinspectionsea#1638]
| Subject        | Details                                                                       |
| :------------- | :---------------------------------------------------------------------------- |
| Plugin         | Php Inspections (EA Extended) v4.0.6 |
| Language level | PHP 5.6 |

Current behaviour
--
The following `preg_match` is suggested to instead become a strpos:
```php
preg_match( '[^a-zA-Z0-9]', $domain ) !== false
```

Expected behaviour
--
This regex is checking for any non-alpha-numeric character within the `$domain` string. It can't be simply converted to the following suggested code:
```php
0 === strpos($domain, 'A-Za-Z0-9')
```

Environment details
---
Running PhpStorm 2020.2.2 on MacOS 10.15.6

**Comment by ea-inspections-team** [comment]
Thanks for reporting @JasonTheAdams! The regex boundaries are not defined (e.g. '/[^a-zA-Z0-9]/'), so the inspection treats [malformed](https://3v4l.org/PE57L) regex as regular text. Closing the issue.
